springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po

```diff
@@ -33,10 +34,10 @@ msgstr ""
 
 #: springboard_iogt/utils.py:26
 msgid "Your Rights"
-msgstr ""
+msgstr "حقوقك"
 
 #: springboard_iogt/utils.py:27
-msgid "Free legal information & support"
+msgid "Know and use them!"
```

Comment:
This PR can still land though, but we'll have to do another PR with the new translations for this sentence